Lib/test/test_sysconfig.py

```diff
@@ -413,7 +414,10 @@ def test_library(self):
         else:
             self.assertTrue(library.startswith(f'libpython{major}.{minor}'))
             self.assertTrue(library.endswith('.a'))
-            self.assertTrue(ldlibrary.startswith(f'libpython{major}.{minor}'))
+            if "--enable-framework" not in config_args:
+                self.assertTrue(ldlibrary.startswith(f'libpython{major}.{minor}'))
+            else:
+                self.assertTrue(ldlibrary.endswith(f'{major}.{minor}/Python'))
```

Comment:
Perhaps this would be better structured as a positive test as is done elsewhere in bother the standard library and in other tests, i.e.:

`if sys.platform == 'darwin' and sys._framework:`

Reply:
Thanks. Was unaware of `sys._framework`. I just went looking for clear indications of a framework build in sysconfig vars.

Reply:
The way the `Makefile` looks for framework builds is by checking if `PYTHONFRAMEWORKDIR` is `no-framework`, but I think `sys._framework`, which contains the value of `PYTHONFRAMEWORK`, is good enough.